js/main.js

```diff
@@ -78,4 +95,18 @@ req.onreadystatechange = function() {
     error("An unknown error occured.", "This embedded Scratch profile can not be shown.<br><a href='https://locness3.github.io/scratch-profile-embed/landing' class='btn' target='_blank' rel='noopener'>About Scratch Profile Embed</a>");
   }
 };
-  
+
+// second request for featured project
+
+//var reqFP = new XMLHttpRequest();
+//reqFP.open("GET", 'https://cors-anywhere.herokuapp.com/scratch.mit.edu/site-api/users/all/' + username);
+//reqFP.send();
+//reqFP.onreadystatechange = function() {
+//  if (reqFP.readyState === 4 && reqFP.status === 200) {
+//    var parsedrespFP = JSON.parse(reqFP.responseText);
+//    console.log(parsedrespFP.featured_project);
+//    featuredImg.src = "https://cdn2.scratch.mit.edu/get_image/project/" + parsedrespFP.featured_project + "_282x210.png";
+//    featuredA.href = "https://scratch.mit.edu/projects/" + parsedrespFP.featured_project;
+//    featuredTitle.innerHTML = parsedrespFP.featured_project_data.title;
+//  }
+//}
```

Comment:
Everything looks good, but what is the last commented section for?

Reply:
It's the original code of the second request, which is wrapped in the first request's function, but I kept below for... some reason.

Reply:
There doesn't seem to be any reason to keeping it, but if you have a reason, keep it. Otherwise, I approve the merge.

Reply:
Well, please click Approve.